packages/amplify_auth_plugin_interface/lib/src/Password/UpdatePasswordRequest.dart

```diff
@@ -13,21 +13,21 @@
  * permissions and limitations under the License.
  */
 
+
 import './PasswordOptions.dart';
-import 'package:flutter/foundation.dart';
 
 class UpdatePasswordRequest {
   String newPassword;
   String oldPassword;
-  PasswordOptions options;
+  PasswordOptions? options;
 
-  UpdatePasswordRequest({@required this.newPassword, @required this.oldPassword, this.options});
+  UpdatePasswordRequest({required this.newPassword, required this.oldPassword, this.options});
   Map<String, dynamic> serializeAsMap() {
     final Map<String, dynamic> pendingRequest = <String, dynamic>{};
     pendingRequest['newPassword'] = newPassword;
     pendingRequest['oldPassword'] = oldPassword;
     if (options != null) {
-      pendingRequest['options'] = options.serializeAsMap();
+      pendingRequest['options'] = options!.serializeAsMap();
     }
```

Comment:
Same, if options is guaranteed to be non-null, `!` shouldn't be required.

Reply:
options is not guaranteed to be non null.  options is a nullable type.